src/support/assertRender.ts

```diff
@@ -1,10 +1,23 @@
+import { assign } from '@dojo/core/lang';
 import { isHNode, isWNode } from '@dojo/widget-core/d';
-import { DNode } from '@dojo/widget-core/interfaces';
+import { DNode, HNode, WNode } from '@dojo/widget-core/interfaces';
 import AssertionError from './AssertionError';
 import { diff, DiffOptions } from './compare';
 
 const RENDER_FAIL_MESSAGE = 'Render unexpected';
 
+export interface AssertRenderOptions extends DiffOptions {
```

Comment:
I thought about this... we still need to mixin the default, because sometimes you only pass "extra" stuff and we do have stuff that should be passed unless it is overridden... I think it also makes it a bit confusing externally, a sort of nested thing when actually the user isn't exposed to `diff` in any way directly.